com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -435,11 +435,7 @@ int GetDestination()
                     m_Materials.uber.EnableKeyword(ShaderKeywordStrings.UseFastSRGBLinearConversion);
                 }
 
-                if (isSceneViewCamera)
-                {
-                    // Scene-view doesn't necessarily use the final-blit, so ensure we set the debug properties here (if necessary)...
-                    DebugHandler?.UpdateShaderGlobalPropertiesFinalBlitPass(cmd, ref cameraData);
-                }
+                GetActiveDebugHandler(renderingData)?.UpdateShaderGlobalPropertiesForFinalValidationPass(cmd, ref cameraData, !m_HasFinalPass);
```

Comment:
Why is the scene view camera check not needed anymore?

Reply:
This fixes an issue where DEBUG_DISPLAY keyword was enabled for parts of the pipeline that it shouldn't be. There are multiple passes that can potentially be the "actual last pass rendered". We need to hook the pixel debug validation logic into exactly this last pass (enabling DEBUG_DISPLAY keyword), and make sure this keyword is disabled from all other "potentially final passes", to avoid having the logic active in multiple passes.

Now all of these potentially last passes (FinalBlitPass, UberPost, "Final Pass") call this function, which can either enable or disable the keyword based on whether the pass is actually final. The previous check was trying to do the same thing, but it didn't work right.